server/routes/createReport.js

```diff
@@ -116,13 +137,18 @@ module.exports = function NewIncidentRoutes({ reportService, offenderService, in
     const involvedStaff =
       (input && input.involvedStaff) || (formId && (await involvedStaffService.getDraftInvolvedStaff(formId))) || []
 
+    const prison = await locationService.getPrisonById(token, prisonId)
+
     const data = {
       ...input,
       displayName,
       offenderNo,
       incidentDate: getIncidentDate(incidentDate, input && input.incidentDate),
       locations,
       involvedStaff,
+      prison,
+      editMode,
+      bookingId,
```

Comment:
I'm not sure why we would need to add edit mode and booking id here? I would have thought they already existed?

Reply:
if you look at render form - they are added there I think?

Reply:
as an aside - I definitely think that we should look at extracting out an edit incident details page soon.. trying to handle most of this stuff in a generic way is getting more and more complicated and incident details is the main culprit

Reply:
removed